test/integration/lib/config.ts

```diff
@@ -41,7 +41,9 @@ const loadEnv = (filePath: string): EnvConfig => {
   })
 
   if (missingKeys.length > 0) {
-    const errorMessage = `Missing required environment variable(s): ${missingKeys.join(', ')}`
+    const errorMessage = `Missing required environment variable(s): ${missingKeys.join(
+      ', '
+    )}`
```

Comment:
this was changed as a result of your formatter?

Reply:
oh, yes it was.